src/components/ReactGoogleMaps/ReactGoogleMaps.module.scss

```diff
@@ -1,6 +1,6 @@
 .searchBarContainer {
   position: relative;
-  top: 57px;
+  top: 608px;
```

Comment:
I'm not 100%  since I don't have this running locally but you might want to do this from the bottom instead of top or you might get inconsistent results based on the screen sizes

Reply:
Thanks for the feedback. My most recent commit does it from the bottom now